### PR TITLE
[Not for review]check in the scripts

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: eng/common/scripts/Verify-Links.ps1 [hunk 1]
```diff
@@ -134,88 +133,135 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
...
+      $replacedOne = $replacedOne -replace "file:///C:/sdk/azure-sdk-for-python", "https://github.com/Azure/azure-sdk-for-python/blob/master"
+      LogWarning "DO NOT use relative link $linkUri. Suggest Link: $replacedOne."
       $linkValid = $false
     }
-     # Check if link uri includes locale info.
-    if ($linkUri -match $locale) {
-      LogWarning "DO NOT include locale $locale information in links: $linkUri. Check here for more information: https://aka.ms/azsdk/guideline/links"
-      $linkValid = $false
+    #  # Check if link uri includes locale info.
+    # if ($linkUri -match $locale) {
```
Comment (weshaggard) [inline eng/common/scripts/Verify-Links.ps1:215]: Might also be worth leaving this check on and fixing any more of these you might find.

Comment (sima-zhu) [reply eng/common/scripts/Verify-Links.ps1:215]: The locale stuff fixed together on the start of the script to save the operations.
https://github.com/Azure/azure-sdk-for-python/pull/13847/files#diff-e9e5d0400c5b5662fb38c88a2d65b0bcR341

Comment (sima-zhu) [reply eng/common/scripts/Verify-Links.ps1:215]: We use the original one to test if any broken links left. By far, it seems enough

**Comment by weshaggard** [comment]
Closing this now that we are done.

**Pull request closed without merging**